src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -286,6 +307,23 @@ const Notebooks = _.flow(
       }))
     )(notebooks)
 
+    const getGalaxyText = () => {
+      return app ?
+        div({ style: { fontSize: 18, lineHeight: '22px', width: 160 } }, [
+          div(['Galaxy Interactive']),
```

Comment:
For the benefit of screen readers, we should try to merge these `div`s, inserting linebreaks if we want particular formatting. (same with "Create a Cloud Environment for Galaxy" below)

Reply:
I'm not seeing this be an issue with ANDI or AXE, how did you discover the screenreader had trouble with this?

Reply:
Ah, I didn't exactly test it. I'm assuming that a screenreader will think that the separate divs are separate pieces of content. It would be good to verify that. Time to relearn VoiceOver. :\